django-stubs/contrib/admin/options.pyi

```diff
@@ -227,9 +247,9 @@ class ModelAdmin(BaseModelAdmin):
     def history_view(self, request: HttpRequest, object_id: str, extra_context: None = ...) -> HttpResponse: ...
 
 class InlineModelAdmin(BaseModelAdmin):
-    model: Any = ...
-    fk_name: Any = ...
-    formset: Any = ...
+    model: Type[Model] = ...
+    fk_name: str = ...
+    formset: BaseFormSet = ...
```

Comment:
I think this should be: `formset: Type[BaseFormSet] = ...`

Reply:
@sssbox, please submit a PR if you think that this is correct.